### DISCUSSION TITLE
Location Maps

### DISCUSSION THREAD

**Discussion by nocturnlbeing** [pretix/pretix#2665] [Q&A] [answered]
New to pretix. Have it installed on VPS.  Need to know if it's possible to show a map of the event location? I've tried adding geo-location co-ordinates / Opencage & Mapquest api keys but they don't seem to work (I am guessing they are for another feature). Can someone please advise and/or supply a work around?  Displaying a location map is more or less a basic feature of these type of programs. Thanks in advance.

**Comment by raphaelm** [comment, marked as answer]
see #1939 and #2524: no, this is currently not possible.
